sdks/node.md

```diff
@@ -11,18 +13,43 @@ const BandwidthMessaging = require('@bandwidth/messaging');
 BandwidthMessaging.Configuration.basicAuthUserName = "token";
 BandwidthMessaging.Configuration.basicAuthPassword = "secret";
 var messagingController = BandwidthMessaging.APIController;
+
+const BandwidthVoice = require('@bandwidth/voice');
+BandwidthVoice.Configuration.basicAuthUserName = "username";
+BandwidthVoice.Configuration.basicAuthPassword = "password";
+var voiceController = BandwidthVoice.APIController;
+
+const BandwidthBxml = require('@bandwidth/bxml');
```

Comment:
`let`'s be consistent. Replace `var` with `const` where applicable and `var` with `let` when mutability needed

Reply:
I guess should they all be `const` then? Since you won't be mutating any of them really